propolis-server/src/config.rs

```diff
@@ -0,0 +1,71 @@
+use std::collections::{btree_map, BTreeMap};
```

Comment:
Here's an example config file I'm using:

```
bootrom = "/export/home/smklein/OVMF_CODE.fd"

[dev.block0]
driver = "pci-virtio-block"
disk = "/export/home/smklein/alpine2.iso"
pci-path = "0.4.0"

[dev.net0]
driver = "pci-virtio-viona"
vnic = "vnic_prop0"
pci-path = "0.5.0"
```

It's somewhat of a kludge; I expect the devices to eventually removed as we migrate to a distributed device.